src/amo/components/HomeHeroBanner/styles.scss

```diff
@@ -42,33 +42,35 @@ $colors-third-item: ('cyan', #15aabf, #1098ad) ('teal', #12b886, #0ca678);
 }
 
 .HeroSection {
-  &:nth-child(1) {
-    $color: nth($colors-first-item, 1);
-    $bg-color: nth($color, 2);
-    $fg-color: nth($color, 3);
+  &.HeroSection-styleName--default {
```

Comment:
I would not make this change, instead you can update the class name L44. There is no need to add yet another nested level. See my suggestion above.